vagrant/ansible/roles/ctdb.setup/tasks/main.yml

```diff
@@ -35,7 +35,6 @@
   firewalld:
     service: ctdb
     permanent: yes
-    state: present
```

Comment:
This hunk is correct. There are two aspects to the fix:
- Specifying `state` twice is not correct. If you want to do it, you need to loop over it (or duplicate the block).
- `present` is not applicable here. See https://docs.ansible.com/ansible/latest/modules/firewalld_module.html . This should ideally be reflected in the commit message or PR message.